editor/import/resource_importer_scene.h

```diff
@@ -67,6 +67,9 @@ class EditorSceneFormatImporter : public RefCounted {
 		IMPORT_GENERATE_TANGENT_ARRAYS = 8,
 		IMPORT_USE_NAMED_SKIN_BINDS = 16,
 		IMPORT_DISCARD_MESHES_AND_MATERIALS = 32, //used for optimizing animation import
+		IMPORT_DISCARD_TEXTURES = 64,
```

Comment:
I added the possibility for `EditorSceneFormatImporter` to supply import options, so please avoid this hack and use this.

Reply:
EditorSceneFormatImporter uses import options, which then feed into the gltf document options, will need to change the  gltf document api.